en-US/focus-ios.xliff

```diff
@@ -199,12 +199,10 @@
       <trans-unit id="Autocomplete.customDescriptoin">
         <source>Add and manage custom autocomplete URLs.</source>
         <target>Add and manage custom autocomplete URLs.</target>
-        <note>Description for adding and managing custom autocomplete URLs</note>
```

Comment:
These removals are due to [these changes](https://github.com/mozilla-mobile/focus-ios/pull/1509/files#diff-63e46856b9849dcaaa62b3c593b8947f). Please let me know if I need to add these redundant strings back, but they are no longer being used.